code/modules/mapping/reader.dm

```diff
@@ -304,6 +306,12 @@
 	//Instanciation
 	////////////////
 
+	for (var/_turf in turf_blacklist)
+		var/turf/turf_in_blacklist = _turf
+		if (crds == turf_in_blacklist) //if the given turf is blacklisted, dont do anything with it
+			turf_blacklist -= turf_in_blacklist
```

Comment:
Or remove it ? If the list is lengthy this can be costly, and there's no reason to since reader won't process same turf twice.

Reply:
no longer removes it